spec/models/subscriber_spec.rb

```diff
@@ -41,5 +41,19 @@
 
       expect(subject).to be_invalid
     end
+
+    it "is invalid if an email address is already taken" do
+      FactoryGirl.create(:subscriber, address: "foo@bar.com")
+
+      subject.address = "foo@bar.com"
+      expect(subject).to be_invalid
+    end
+
+    it "is case-insensitive when validating if an address is taken" do
+      FactoryGirl.create(:subscriber, address: "FOO@bar.com")
+
+      subject.address = "foo@BAR.COM"
+      expect(subject).to be_invalid
```

Comment:
Technically, these are distinct email addresses. The part before the @ is case sensitive, the part after the @ is not.

Reply:
(massive edge case, and most servers tend to be case insensitive on the part before @ too, so maybe it doesn't matter)

Reply:
Ahh – I didn't know that. I think we shouldn't restrict it in that case. It's simpler without, anyway.

Reply:
I suppose we could write our own validation which checks the part after the @ being case sensitive and the part before being case insensitive, but that feels a bit extreme.